lib/hex_web/web/templates/docs/publish.html.eex

```diff
@@ -1,13 +1,13 @@
 <h2>Publishing a package</h2>
 
 <p>
-  Publishing a package to Hex consists of creating a user, attaching some metadata to the project's <code>mix.exs</code> file and finally performing the command to push.
+  Publishing a package to Hex consists of registering a Hex user, adding metadata and dependencies to the project's <code>mix.exs</code> file, and finally submitting the package with a <code>mix</code> task.
```

Comment:
I think we can assume the user already added dependencies, otherwise the project wouldn't work.